src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ProcurementEventService.java

```diff
@@ -117,8 +119,16 @@ public EventSummary createEvent(final Integer projectId, final CreateEvent creat
       var existingEvent = project.getProcurementEvents().stream().iterator().next();
       terminateEvent(projectId, existingEvent.getEventID(), TerminationType.CANCELLED, principal);
 
-      var rfxResponse = jaggaerService.getRfx(existingEvent.getExternalEventId());
-      suppliers = rfxResponse.getSuppliersList().getSupplier();
+      if (existingEvent.isTendersDBOnly()) {
```

Comment:
I added this block of code and 170 line. The rest of the differences are format changes.